packages/immer/src/immer.ts

```diff
@@ -0,0 +1,23 @@
+import { Atom } from '@frp-ts/core'
+import { mapRecord } from '@frp-ts/utils'
+import immer, { Draft } from 'immer'
+
+interface Recipe<Value> {
+	// any is needed due to variance
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	(...args: readonly any[]): (draft: Draft<Value>) => void
```

Comment:
We can use the [immer](https://github.com/immerjs/immer/blob/fcb7a5590ce46d7dddee6f5382fb0f366370b95d/compat/pre-3.7/dist/immer.d.ts#L93
) syntax for this recipe

Reply:
It's simpler, see [doc](https://immerjs.github.io/immer/curried-produce)
```ts
{
  toggleTodo: (draft, id) => {
      const todo = draft.find(todo => todo.id === id)
      todo.done = !todo.done
  }
}
```